README.md

```diff
@@ -25,6 +25,7 @@ This collection provides a list of educational resources for Software Engineers.
 - [CodeChef](https://www.codechef.com/) - Non-profit competitive programming platform.
 - [Topcoder](https://www.topcoder.com/) - Participate in code challenges and help solve real-world problems.
 - [Codeforces](https://codeforces.com/) - Russian website dedicated to competitive programming.
+- [URI Online Judge](https://www.urionlinejudge.com/judge/en/login) - The main goal of the project is to provide programming practice and knowledge sharing.
```

Comment:
Please can you remove the competitive programming link you added from here, and leave it in the competitive programming separate file. This is because we only added 5 to our preview and you can see the rest of the items in the separate md file. Thank you.

Reply:
Nice catch @CodingSpecies